README.md

```diff
@@ -175,6 +175,8 @@ The minimum severity level can be set using the `--minSeverity/-m` flag.
 
 By default kubeaudit will output results in a human-readable way. If the output is intended to be further processed, it can be set to output JSON using the `--format json` flag. To output results as logs (the previous default) use `--format logrus`. Some output formats include colors to make results easier to read in a terminal. To disable colors (for example, if you are sending output to a text file), you can use the `--no-color` flag.
 
+You can generate a kubeaudit report in [SARIF](https://docs.oasis-open.org/sarif/sarif/v2.0/sarif-v2.0.html) and write it to a file by using the `-s/--sarif` flag.
```

Comment:
Good call. Okie. I checked `semgrep` as an example and it does print sarif to the screen and you can specify formats like `--sarif` and `json`

So we can do:
`semgrep scan --config auto --sarif . > my.sarif`  if we want to save it. So, I'll refactor to print instead of save the file directly

Reply:
addressed here: https://github.com/Shopify/kubeaudit/pull/453/commits/a386749ae5253e6cef9d998a495f8768a6474e40